pkg/shell/index.html

```diff
@@ -360,7 +360,7 @@ <h4 class="modal-title" translatable="yes">Authentication</h4>
                     </div>
                 </div>
                 <div class="has-feedback search" id="menu-search">
-                    <input id="filter-menus" class="form-control" type="search" placeholder="Type to search..." translate="placeholder">
+                    <input id="filter-menus" class="form-control" type="search" placeholder="Search" translate="placeholder">
```

Comment:
Axe gave me a critical error that this didn't have a label.

Reply:
> Axe gave me a critical error that this didn't have a label.

Hups. This is though not related to this PR as @garrett mentioned.

> Also, shouldn't these be translated?

It is translated, notice the `translate="placeholder"`

Reply:
Aha! Can translate also translate aria-label _and_ placeholder at the same time? (Does it accept more than one value, that is...)